src/lib_json/json_value.cpp

```diff
@@ -16,6 +16,7 @@
 #include <iostream>
 #include <sstream>
 #include <utility>
+#include <math.h>
```

Comment:
why do we need both cmath and math.h?

Reply:
Because Embarcadero C++Builder 10.3 complains that doesn't know about modf().
Maybe it could be improved with an #ifdef

Reply:
What if we changed our `modf` call to `std::modf`? Would that help?
https://github.com/open-source-parsers/jsoncpp/blob/f2f5ce866ba4b0286b5d6ec7f595b5c0d6989381/src/lib_json/json_value.cpp#L1255

Reply:
Indeed! It works with std::modf.
So #include <math.h> is not needed anymore.